unity/Assets/Maroon/GlobalEntities/SoundManager/SoundManager.cs

```diff
@@ -66,6 +66,14 @@ public class SoundManager : MonoBehaviour, GlobalEntity
         public static SoundManager Instance => SoundManager._instance;
         MonoBehaviour GlobalEntity.Instance => Instance;
 
+        // -------------------------------------------------------------------------------------------------------------
+        // Audio sources
+        
+        /// <summary>
+        ///     Getter for Sound Effect AudioSource component
+        /// </summary>
+        public AudioSource soundEffectSource => _soundEffectSource;
```

Comment:
Added a getter to access SoundManager's sound effect AudioSource component from other prefabs to unify all sound effects to one source.